googletest/include/gtest/gtest.h

```diff
@@ -2116,9 +2116,11 @@ GTEST_API_ AssertionResult DoubleLE(const char* expr1, const char* expr2,
 // of the dummy variable name, thus allowing multiple SCOPED_TRACE()s
 // to appear in the same block - as long as they are on different
 // lines.
-#define SCOPED_TRACE(message) \
-  ::testing::internal::ScopedTrace GTEST_CONCAT_TOKEN_(gtest_trace_, __LINE__)(\
-    __FILE__, __LINE__, ::testing::Message() << (message))
+#define SCOPED_TRACE_(message, file, line) \
```

Comment:
I think it should be (file, line, message) rather than (message, file, line). This will allow for variadic extension of message.... Can we do something clever with __VA_ARGS__ to implement both SCOPED_TRACE variants under the same name?